RFC/src/Glossary.md

```diff
@@ -184,6 +189,11 @@ A Mining Server is responsible for constructing new blocks by bundling transacti
 
 A Mining Worker is responsible for performing Proof-of-Work tasks received from its parent [Mining Server].
 
+## Multisig
+[multisig]: #multisig
+
+Multi-signatures (Multisigs) are also known as N-of-M signatures, and this means that N amount out of M amount of peers need to agree before a transaction can be spend. N and M can also be the same amount.
```

Comment:
Consider changing "signatures, and this means that N amount out of M amount of peers need" to "signatures, this means that a minimum of N number of the M peers need"